grails-app/views/purchaseOrderWorkflow/purchaseOrder/showOrderItems.gsp

```diff
@@ -320,6 +321,8 @@
             success: function () {
               clearOrderItems();
               loadOrderItems();
+              $('#orderItems').html('<option></option>').trigger('change');
```

Comment:
What is this empty option for? I think that noSelection functions as an empty option.

Reply:
It's used on item deletion to clear options in select. Otherwise, deleted item is still visible in dropdown because select doesn't remove it.

Reply:
Actually I noticed that when I add items and then go to the adjustments tab the Order Items dropdown is empty. Which makes sense because we don't explicitly refresh it. You have to refresh the page in order for it to populate. Can you test whether this refresh would refresh under that use case as well? It seems it should but if not let's try to find a way to refresh it.

Reply:
@jmiranda Refresh of the page helps in both cases - adding and deleting items. In this PR I have added a way to refresh this select after adding/deleting items. It work for both cases but in case of deleting I had to add this line above to clear options first.